source/web/src/components/filesystems.vue

```diff
@@ -41,8 +49,9 @@ export default {
               this.filesystems = response
           }
           catch (error) {
```

Comment:
My concern with this is that the catch block is really broad, meaning any exception has the potential to change the state of the filesystems. 

For example, let's say there is a valid error when retrieving the filesystems (maybe a 500 error because lambda is having issues), instead of accurately catching that error, we're instead defaulting to just saying there are no filesystems.

We should make this more robust by parsing the error response to determine exactly what happened, then render the page conditionally based on that.

Reply:
Sounds good. 

I will look at the responses returned and see which one signifies no file systems are available.